apps/prairielearn/src/question-servers/freeform.js

```diff
@@ -494,7 +494,7 @@ function checkData(data, origData, phase) {
        || checkProp('answers_names',         'object',  ['prepare'],                          ['prepare']);
   if (err) return err;
 
-  const extraProps = _.difference(_.keys(data), checked);
+  const extraProps = _.difference(Object.keys(data), checked);
```

Comment:
We'll get [native `Set.difference`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/difference) in Node 22!